test/beer-router-test.js

```diff
@@ -0,0 +1,193 @@
+      .then(brewery => {
+        this.tempbrewery = brewery;
+        return new Beer({
+          nameOfBeer: 'whatever',
+          typeOfBeer: 'stout',
+          percentOfBeer: 8.9,
+          breweryID: this.tempbrewery._id.toString(),
+        }).save()
+        .then(beer => {
+          this.tempbeer = beer;
+          done();
+        })
+        .catch(done);
+      });
+      it('should rspond with a beer', (done) => {
+        superagent.get(`${baseURL}/api/beers/${this.tempbeer._id.toString()}`)
+        .then(res => {
+          expect(res.status).to.equal(200);
+          expect(res.body.breweryID).to.equal(this.tempbrewery._id.toString());
+          expect(res.body.nameOfBeer).to.equal('whatever');
+          expect(res.body.typeOfBeer).to.equal('stout');
+          expect(res.body.percentOfBeer).to.equal(8.9);
+          done();
+        })
+        .catch(done);
+      });
+      it('should return a 404 error on an invalid id', (done) => {
+        superagent.get(`${baseURL}/api/beers/3456`)
+        .then(done)
+        .catch(err => {
+          expect(err.status).to.equal(404);
+          done();
+        })
+        .catch(done);
+      });
+    });
+
+    describe('testing DELETE /api/beers/:id', function() {
+      beforeEach(done => {
+        new Brewery({nameOfBrewery: 'random'}).save()
+        .then(brewery => {
+          this.tempbrewery = brewery;
+          new Beer({
+            nameOfBeer: 'whatever',
+            typeOfBeer: 'stout',
+            percentOfBeer: 8.9,
+            breweryID: this.tempbrewery._id.toString(),
+          }).save()
+          .then(beer => {
+            this.tempbeer = beer;
+            done();
+          })
+          .catch(done);
+        });
+        it('should give a 204 status with a valid id', (done) => {
+          superagent.delete(`${baseURL}/api/beers/${this.tempbeer._id}`)
+          .then(res => {
+            expect(res.status).to.equal(204);
+            done();
+          })
+          .catch(done);
+        });
+        it('should give a 404 status with an invaild id', (done) => {
+          superagent.delete(`${baseURL}/api/beer/34566`)
+          .then(done)
+          .catch(err => {
+            expect(err.status).to.equal(404);
+            done();
+          })
+          .catch(done);
+        });
+      });
+
+//BOUNS
+      describe('test GET /api/beers', function() {
+        beforeEach(done => {
+          new Brewery({nameOfBrewery: 'random'}).save()
+          .then(brewery => {
+            this.tempbrewery = brewery;
+            new Beer({
+              nameOfBeer: 'whatever',
+              typeOfBeer: 'stout',
+              percentOfBeer: 8.9,
+              breweryID: this.tempbrewery._id.toString(),
+            }).save()
+            .then(beer => {
+              this.tempbeer = beer;
+              done();
+            })
+            .catch(done);
+          });
+
+          it('should return all beers', (done) => {
```

Comment:
Great job creating and testing your routes for getting all the beers and breweries!